driver/src/orderbook/streamed/orderbook.rs

```diff
@@ -334,7 +342,7 @@ mod tests {
         orderbook.handle_event_data(withdraw_request, 1, 0, H256::zero(), 0);
         orderbook.handle_event_data(token_listing_0, 0, 0, H256::zero(), 0);
 
-        let auction_data = orderbook.get_auction_data(2.into()).unwrap();
+        let auction_data = orderbook.get_auction_data(2.into()).wait().unwrap();
```

Comment:
Maybe for tests we should use [`FutureExt::now_or_never`](https://docs.rs/futures/0.3.5/futures/future/trait.FutureExt.html#method.now_or_never)?